package.json

```diff
@@ -70,7 +70,7 @@
     "resolve": "^1.3.3"
   },
   "engines": {
-    "node": "^4.5 || 6.* || >= 7.*"
+    "node": "^4.5 || 6.* || >= 7.* || >= 8.*"
```

Comment:
Note that message is talking about `ember-cli-babel@6.1.0` which had `|| 7.*` not `|| >= 7.*`.  You just need to update your minimum version to incorporate the fix from https://github.com/babel/ember-cli-babel/pull/155.